lib/as_action_ocaml.ml

```diff
@@ -67,7 +67,7 @@ let compute_deps_ml = compute_deps `Ml
 
 let compile_mli
     ?(needs = []) ?(pkgs = nop) ?(args = []) ~ocamlc ~annot ~incs ~src () =
-  let ctx = As_ctx.v [`OCaml; `Compile; `Src `Mli] in
+  let ctx = As_ctx.v [`OCaml; `Compile; `Src `Mli; `Target `Byte] in
```

Comment:
Note also that the `ocamlc` argument will be `ocamlopt` if the byte code compiler is not available see here https://github.com/samoht/assemblage/blob/master/lib/as_part_unit.ml#L128-L129